modules/react/common/stories/Common.mdx

```diff
@@ -0,0 +1,103 @@
+import {ExampleCodeBlock} from '@workday/canvas-kit-docs';
+
+import {BaseExample} from './mdx/examples/BaseExample';
+
+
+<Meta title="Hooks and Utilities/Common" />
+
+# Common
+The Common package contains a collection of components and utilities that serve as the foundation for building robust features across other packages.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Base
+
+The `Base` component is a low level component that automatically handles the [`cs`](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--docs#cs-prop) prop for you and style merging via [`handleCsProp`](https://workday.github.io/canvas-kit/?path=/docs/styling-merging-styles--docs#handlecsprop).
+As we transition away from runtime styles, you should use `Base` when styling an element where you want more control over the styles and a Canvas Kit component doesn't quite fit your needs.
+
+### Usage
+
+Use `Base` when you need to statically style a container element or you're building a complex component that is **not reusable**.
+
+
+```tsx
+import {Base} from '@workday/canvas-kit-react/common';
+import {system} from '@workday/canvas-tokens-web';
+import {createStyles} from '@workday/canvas-kit-styling';
+
+const headerStyles = createStyles({
+	fontSize: system.fontSizes.xl,
+	fontWeight: system.fontWeights.bold,
+	color: system.color.text.primary,
+});
+
+const containerStyles = createStyles({
+	display: 'flex',
+	flexDirection: 'column',
+	gap: system.space.x2,
+})
+
+// Base+cs
+const AppHeader = ({children, headerText, other, props}) => {
+
+  return (
+    <Base cs={containerStyles}>
+      <Base as="h2" data-testid="my-header" cs={headerStyles}>{headerText}</Base>
+      <Base as='p' cs={someOtherStyles}>
+        // ...
+      </Base>
+    </Base>
+  )
+}
+```
```

Comment:
Do you think this code snippet adds anything beyond the Basic Example below? If not, we could delete it and just have the example.